chore/ci-checkstyle-javadoc.sh

```diff
@@ -72,6 +83,8 @@ function main() {
         echo "Javadoc quality has deteriorated!"
         echo "Run the chore/ci-checkstyle-javadoc.sh script locally to find errors"
         echo "See $RELATED_ISSUE_URL for details"
+        echo -e "\n"
+        python3 chore/find_javadoc_regression.py "$OUTPUT_FILE_COMPARE" "$OUTPUT_FILE_OWN"
```

Comment:
Here's a good reason why we'd want to compose this whole thing into Python: the name of the Python executable is something of a moving target right now. Ever since Python 2 went belly-up, the conventions have been in flux. Some systems now only have `python`.

We want it to be possible to run this script locally with as little effort as possible.

Reply:
Your `ci-extra` script actually broke in a docker container I tried it in because the ubuntu latest container symlinked `python` to `python2` and your scripts require python 3. That's why I opted for `python3` as the executable name here.

But yea, the whole situation is a bit unfortunate...

Reply:
Well, `ci-extra.sh` requiring a little bit of tinkering to get running is fine IMO, as it's not meant to run locally. But as this script is meant to be run locally both to fix regressions and to actually improve the Javadoc quality as part of #3923, we'll want to make it as hassle-free as humanly possible.

Python 3.6+ should arguably be more hassle-free than bash, as the former also works on (native) Windows.